getting-started/installation-homebrew.md

```diff
@@ -2,9 +2,7 @@
 
 This will install both TimescaleDB *and* PostgreSQL via Homebrew.
 
-**Note: TimescaleDB requires PostgreSQL 11.4+ or 12.0+. Support for
-PostgreSQL 9.6.3+ and 10.9+ is deprecated and will be removed in a
-future release.**
+**Note: TimescaleDB requires PostgreSQL 11 or 12.**
```

Comment:
Why not clarify 11.4+?

Reply:
because its 11.0+ not sure why there is sometimes 11.4 mentioned but its not a requirement and our CI tests 11.0